INSTALL.md

```diff
@@ -83,12 +83,14 @@ your `.zshrc`.
     git clone https://github.com/zsh-users/zsh-syntax-highlighting.git ${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-syntax-highlighting
     ```
 
-2. Activate the plugin in `~/.zshrc`:
+2. Activate the plugin **at the bottom** of `~/.zshrc`:
 
     ```zsh
     plugins=( [plugins...] zsh-syntax-highlighting)
     ```
 
+See the [FAQ](https://github.com/zsh-users/zsh-syntax-highlighting#why-must-zsh-syntax-highlightingzsh-be-sourced-at-the-end-of-the-zshrc-file) for more details.
```

Comment:
Is it possible to use a relative URL here? The documentation gets installed and should be usable offline.